lib/smart_answer_flows/business-coronavirus-support-finder/business_coronavirus_support_finder.erb

```diff
@@ -7,6 +7,8 @@
 <% end %>
 
 <% govspeak_for :body do %>
+  <%= render 'smart_answers/shared/country-notice' %>
```

Comment:
Just FYI - start pages are rendered by Frontend as "Transaction pages". This view is only used for development purposes.

Reply:
Didn't know this, thanks @theseanything